pkg/shell/templates/change-auth.html

```diff
@@ -22,7 +22,7 @@ <h4 class="modal-title" translatable="yes">Log in to {{host}}</h4>
                 <td>
                     <a tabindex="0" role="button" data-toggle="popover"
                         data-trigger="focus" data-placement="bottom"
-                        data-content="Leave blank to connect to this machine as the currently logged in user ({{user}}). If you enter a different username, that user will always be used connecting to this machine.">
+                                             data-content="Leave blank to connect to this machine as the currently logged in user {{#user}}({{user}}){{/user}}. If you enter a different username, that user will always be used connecting to this machine.">
```

Comment:
why the whitespace change?
Also, won't this text look strange if user is null / empty? You could add an "else" mustache part or move the space into the `user` conditional section.